src/controller/java/MdnsImpl.cpp

```diff
@@ -47,6 +47,11 @@ CHIP_ERROR ChipMdnsInit(MdnsAsyncReturnCallback initCallback, MdnsAsyncReturnCal
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR ChipMdnsShutdown()
+{
+    return CHIP_NO_ERROR;
+}
+
```

Comment:
It's already in namespace `chip::Mdns`. Sounds like what we need is `Shutdown` method, `Init()`, method, etc.

Reply:
That naming makes sense to me; however, it's not strictly related to this change, and there are a couple of open PRs for which it would cause merge conflicts.